_posts/Illumination_Correction_A_Comparison_of_Methods.md

```diff
@@ -0,0 +1,135 @@
+I needed to use this conversion because downloading the images as-is (without conversion) was causing multiple errors during my downstream processes. 
+It is important to know that you will likely have to retrace your steps back to previous pipelines, like IC, when issues need to be corrected or to be improved. 
+In all, I believe this method is the most efficient and easy to work with to perform illumination correction.
+
+Lastly, I checked out the Fiji/ImageJ plugin called CIDRE. 
+I don’t have a lot to say about it since when I used it with my image set of 96 images, it came up with an error. 
+After further investigation, it was [an error](https://github.com/smithk/cidre/issues/3) that was found back in 2018 that has not been solved. Though, from an outsider’s perspective, this could seem like the developers abandoned the project, but this might not be true. 
+I have started to understand the challenges of maintaining open-source software through my position.
+To all those that do, I thank you for your hard work! 
+It is unfortunate that I could not test this method to determine how well it worked with my images. 
+I have determined that this method, though referenced a lot during my research on the topic, has not been improved upon and likely not a valid option for illumination correction at this current time. 
+I will investigate this method in the future and I hope to see it be improved upon!
+
+## Conclusion
+
+Based on this information I have provided, I hope that this guides you in the right direction for your illumination correction pipeline. 
+There are many other software/packages/methods that can be used to do illumination correction, but it takes time to figure out the “right” one. 
+For the needs of my current project, I chose PyBaSiC! 
+The package is written in Python (which is interoperable with our current data analysis ecosystem), no manual parameters for determining the illumination correction function, faster to run, and is easy to work with as a new computational biologist. 
+We have yet to determine the impact of this method of illumination correction on the downstream cell morphology readouts, but we plan to test these empirically in the near future (with a blog post of course! 😉).
+
+If I find and investigate any other methods, I will update this blog to provide information and opinions on them.
+
+## Supplementary
+
+To view the progress being made with the NF1 Schwann Cell project, you can go to GitHub repository to view each pipeline and rationale (see https://github.com/WayScience/NF1_SchwannCell_data).
```

Comment:
I think you should reference the nature of this project sooner... I'll try to find a good spot above.

Reply:
@gwaybio I am referencing it now in the first section